crypto/include/aes_icm_ext.h

```diff
@@ -58,4 +61,22 @@ typedef struct {
     EVP_CIPHER_CTX *ctx;
 } srtp_aes_icm_ctx_t;
 
+#endif /* OPENSSL */
+
+#ifdef NSS
+
+#include <pk11pub.h>
+
+typedef struct {
+    v128_t counter;
+    v128_t offset;
+    int key_size;
+    uint8_t iv[16];
+    PK11SlotInfo *slot;
```

Comment:
You shouldn't need this here, as it is only used for the PK11_ImportSymKey call.